hack/observability/grafana/chart/values.yaml

```diff
@@ -88,8 +88,39 @@ dashboardProviders:
       options:
         path: /var/lib/grafana/dashboards/default
 
-dashboardsConfigMaps:
-  default: grafana-dashboards
+dashboardsConfigMaps: {}
```

Comment:
Is there a way to just get rid of that annotation?

Reply:
nope we need that entry afaik.

This maps the directory we mount in the dashboards to the datasource.

Reply:
@chrischdi ^^ (no rush, just not sure if you saw and now the conversation is outdated :))

Reply:
Ah I thought you meant this valuesfile entry.

Well, it uses KubectL apply (tilt) I'm not aware how we could do that as long as we're not using go

Reply:
Was hoping maybe we can tell tilt to deploy with replace or eg enable SSA for apply

Reply:
I don't see a way to do this. (lookedup the api docs for `k8s_yaml` and `k8s_resource`)

Reply:
Alright, thank you!